samples/matter/common/src/binding_handler.cpp

```diff
@@ -5,21 +5,21 @@
  */
```

Comment:
Please modify the license of new files to contain the correct creation date (2022->2023)